src/Helper/Service/AssetFactory.php

```diff
@@ -48,4 +51,18 @@ public function createService(ServiceLocatorInterface $serviceLocator, $rName =
     {
         return $this($serviceLocator, $cName);
     }
+
+    /**
+     * @param array<array-key, mixed> $array
+     * @return array<array-key, mixed>
+     */
+    private function assertArray(string $key, array $array): array
+    {
+        $value = $array[$key] ?? [];
+        if (! is_array($value)) {
+            throw new Exception\RuntimeException('Invalid resource map configuration.');
```

Comment:
This exception doesn't tell us which key is missing and/or malformed - this exception will just be confusing